README.md

```diff
@@ -68,8 +68,10 @@ Finally, with given scenarios and multi-partner learning approaches, we can addr
     n_repeats: 5
     scenario_params_list:
      - dataset_name:
-         - 'mnist'
-         - 'cifar10'
+       'mnist':
+        - 'init'
+       'cifar10':
+        - 'init'
```

Comment:
Is it possible to not put init, and if we do that, by default it will do a random initialization ?

Reply:
And by the way, it is still possible to use the older way to enter dataset then the random initialization is applied. I added the possibility to enter dataset as a dictionary key with an empty value.